This job can be monitored from: https://scruffy.c3se.chalmers.se/d/alvis-job/alvis-job?var-jobid=70058&from=1620947999000
> Modules loaded
> Running run_model.py 1 times
> Running 1 program
2021-05-14 01:20:01.050171: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Traceback (most recent call last):
  File "train_model.py", line 39, in <module>
    X_test, y_test, gender_model = gender()
  File "train_model.py", line 24, in gender
    X_train, X_test, y_train, y_test = data(dataset, image_folder, img_shape, test_size, augumentation, logging)
  File "/cephyr/users/gablindg/Alvis/faces/model/data.py", line 18, in data
    for filename in os.listdir(image_folder):
FileNotFoundError: [Errno 2] No such file or directory: 'resized_96_equal_distribution_pictures_grayscale'
> Counting accuracy for 1 runs:
Traceback (most recent call last):
  File "average_accuracy.py", line 10, in <module>
    accuracy = sum / len(content)
ZeroDivisionError: division by zero
